src/rhos/rhoStat.py

```diff
@@ -59,7 +60,7 @@ def parallel_bootstrap_compact_fp_DEPRECATED(
 
 def parallel_bootstrap_compact_fp(par_, in_, out_, start, end, is_folded=False):
     import rhoUtils
-
+    random.seed(1994)
```

Comment:
Seeing a call to `random.seed` in the middle of a function whose name starts with `parallel` makes me nervous. What's this doing?

Reply:
The random seed declaration here was added by @LupoA: what should we do about it? This question is for both.

Reply:
This function resamples the correlator by bootstrapping. In the resampled set of measurements, each of them comes from averaging a random set of correlators.  The seed should determine which set is averaged. This function is called by the class ParallelBootstrapLoop.

Understanding this is not a safe way of fixing the seed, what would be the way to go, @edbennett ?

Notice that we could also remove the parallelised version, unless one is dealing with >10k measurements this should not be important. There is already a non parallel version.

Reply:
We should move this discussion to another issue. Since the parallel bootstrap is written, I don't see a need to remove it, but we'll probably want to address this issue before releasing.